student/src/test/java/tds/student/services/ReviewTestServiceTest.java

```diff
@@ -0,0 +1,194 @@
+package tds.student.services;
+
+import AIR.Common.Web.TDSReplyCode;
+import AIR.Common.data.ResponseData;
+import TDS.Shared.Exceptions.ReturnStatusException;
+import com.google.common.base.Optional;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.mockito.runners.MockitoJUnitRunner;
+
+import java.util.UUID;
+
+import tds.common.ValidationError;
+import tds.student.services.abstractions.IAdaptiveService;
+import tds.student.services.abstractions.IOpportunityService;
+import tds.student.services.abstractions.IResponseService;
+import tds.student.services.data.PageList;
+import tds.student.services.data.TestOpportunity;
+import tds.student.sql.data.OpportunityInstance;
+import tds.student.sql.data.TestConfig;
+import tds.student.sql.repository.remote.ExamRepository;
+import tds.student.web.TestManager;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Matchers.any;
+import static org.mockito.Matchers.anyBoolean;
+import static org.mockito.Matchers.anyString;
+import static org.mockito.Mockito.doNothing;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.verifyZeroInteractions;
+import static org.mockito.Mockito.when;
+
+@RunWith(MockitoJUnitRunner.class)
+public class ReviewTestServiceTest {
+    @Mock
+    private IResponseService mockResposneService;
+
+    @Mock
+    private IAdaptiveService mockAdaptiveService;
+
+    @Mock
+    private IOpportunityService mockOpportunityService;
+
+    @Mock
+    private ExamRepository remoteExamRepository;
+
+    private OpportunityInstance opportunityInstance = new OpportunityInstance(UUID.randomUUID(),
+        UUID.randomUUID(),
+        UUID.randomUUID(),
+        UUID.randomUUID(),
+        UUID.randomUUID(),
+        "exam-client-name",
+        "browser-agent");
+
+    private TestOpportunity testOpportunity = new TestOpportunity(opportunityInstance,
+        "test-key",
+        "test-id",
+        "language",
+        new TestConfig());
+
+    @Mock
+    private TestManager mockTestManager = new TestManager(testOpportunity,
+        mockResposneService,
+        mockAdaptiveService);
+
+    @Mock
+    private PageList mockPageList = new PageList();
+
+    private ReviewTestService reviewTestService;
+
+    @Before
+    public void setUp() {
+        reviewTestService = new ReviewTestServiceImpl(mockOpportunityService,
+            remoteExamRepository,
+            true,
+            true);
+    }
+
+    @Test
+    public void shouldReviewATestWhenLegacyCallsAndRemoteCallsAreEnabled() throws ReturnStatusException {
+        doNothing().when(mockTestManager).LoadResponses(anyBoolean());
```

Comment:
Rather than use anyBoolean can you use `true` since I believe that is the expected value?  If we know the values that will be passed to the mock I prefer to use the actual values.  Assuming that Mockito can discern the value equality (however it does that).

Same comment for any place you use `any()` where we know the expected value.  We basically want things to fail if you pass in `false` when we expect `true`